app/invocation/invocation_model.tsx

```diff
@@ -261,8 +263,16 @@ export default class InvocationModel {
     return this.invocations.find(() => true).role;
   }
 
+  isBazelInvocation() {
+    return this.getRole() !== CI_RUNNER_ROLE;
+  }
+
   getTool() {
-    return `bazel v${this.started?.buildToolVersion} ` + this.started?.command || "build";
+    if (this.isBazelInvocation()) {
+      return `bazel v${this.started?.buildToolVersion} ` + this.started?.command || "build";
+    }
+
+    return "BuildBuddy CI runner";
```

Comment:
I do think it's a little confusing that in some places in the UI (user facing) these are called workflows, some places they're called actions, and now we're also calling them "CI runs".